client/react/components/signupForm.jsx

```diff
@@ -18,20 +24,38 @@ const SignUpForm = ({
         <Field
             name='email'
             component={LabelInputField}
-            label={{ content: <Icon color='blue' name='user' size='large' /> }}
+            label={{ content: <Icon color='blue' name='mail' size='large' /> }}
             labelPosition='left'
             placeholder='Email'
         />
+        <Field
+            name='confirmEmail'
+            component={LabelInputField}
+            label={{ content: <Icon color='blue' name='mail' size='large' /> }}
+            labelPosition='left'
+            placeholder='Confirm Email'
+        />
         <Field
             name='password'
             component={LabelInputField}
-            label={{ content: <Icon color='blue' name='user' size='large' /> }}
+            label={{ content: <Icon color='blue' name='lock' size='large' /> }}
             labelPosition='left'
             placeholder='Password'
+            type='password'
+        />
+        <Field
+            name='confirmPassword'
+            component={LabelInputField}
+            label={{ content: <Icon color='blue' name='lock' size='large' /> }}
+            labelPosition='left'
+            placeholder='Confirm Password'
+            type='password'
         />
+        <Button type='submit' color='green' loading={isProcessing} disabled={isProcessing}>Join</Button>
```

Comment:
I like the idea of a "Join" button but should we use "Sign Up" for the button text to avoid any confusion for the users?

Reply:
I was thinking Register and Log In/Log Out

Reply:
That works too

Reply:
I agree with @bbaksh